includes/snmp.inc.php

```diff
@@ -57,33 +109,33 @@ function snmp_get_multi($device, $oids, $options = '-OQUs', $mib = null, $mibdir
         $device['transport'] = 'udp';
     }
 
-    $cmd  = $config['snmpget'];
     $cmd .= snmp_gen_auth($device);
+    $cmd .= " $options";
```

Comment:
If $options is null, false, or '' a check isn't needed.  Only if $options is 0, do we need to check it before converting it to a string.  I didn't see it defined as 0 anywhere, and I don't think it would make sense to do that.